src/Utility/CommonOptionsTrait.php

```diff
@@ -45,6 +45,11 @@ trait CommonOptionsTrait
      */
     public $connection;
 
+    /**
+     * @var bool
+     */
+    public $force = false;
```

Comment:
The default value is required because we have some hacky unit tests that call part of the Command instead of baking the file.